models/Discord/Commands/DiscordCommandInfo.js

```diff
@@ -31,10 +31,35 @@ class DiscordCommandInfo extends DiscordCommand {
 					if ('error' in resultsstatus) {
 						return message.reply(results.error);
 					}
-					var round_duration = querystring.parse(resultsstatus.data)["round_duration"]
-					var shuttle_mode = querystring.parse(resultsstatus.data)["shuttle_mode"]
-					var shuttle_time = querystring.parse(resultsstatus.data)["shuttle_timer"]
-					var security_level = querystring.parse(resultsstatus.data)["security_level"]
+					var round_duration = querystring.parse(resultsstatus.data)["round_duration"];
+					var shuttle_mode = querystring.parse(resultsstatus.data)["shuttle_mode"];
+					var shuttle_time = querystring.parse(resultsstatus.data)["shuttle_timer"];
+					shuttle_time = StringUtils.replaceAll(shuttle_time, "\0", "");
+					shuttle_time = shuttle_time/60;
+					var security_level = querystring.parse(resultsstatus.data)["security_level"];
+					switch(shuttle_mode) {
```

Comment:
You could use an array here instead and use the shuttle_mode as an index for the array.